### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jwildfire**
# Overview 

Various updates to documentation. Added v2 vignettes. Rebuilt pkgdown site. No code changes.

# Test Notes

No code changes. Just poke around the new pkgdown site and make notes of any needed changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,54 +2,26 @@
...
-
-This package is being built in conjunction with the [hep-explorer](https://github.com/SafetyGraphics/hep-explorer) javascript library.
+The {safetyGraphics} package provides a framework for evaluating of clinical trial safety in R using a flexible data pipeline. The package includes a shiny application that allows users to explore safety data using a series of interactive graphics, including the hepatic safety explorer shown below.  The package has been developed as part of the <a target="_blank" href="https://safetygraphics.github.io/">Interactive Safety Graphics (ISG) workstream</a> of the <a target="_blank" href="https://community.amstat.org/biop/workinggroups/safety/safety-home">ASA Biopharm-DIA Safety Working Group</a>.
```
Comment (elimillera) [inline README.md:7]:
```suggestion
The {safetyGraphics} package provides a framework for the evaluation of clinical trial safety data in R using a flexible data pipeline. The package includes a shiny application that allows users to explore safety data using a series of interactive graphics, including the hepatic safety explorer shown below.  The package has been developed as part of the <a target="_blank" href="https://safetygraphics.github.io/">Interactive Safety Graphics (ISG) workstream</a> of the <a target="_blank" href="https://community.amstat.org/biop/workinggroups/safety/safety-home">ASA Biopharm-DIA Safety Working Group</a>. 
```

File: vignettes/Cookbook.Rmd [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Cookbook}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Cookbook Vignette
+
+This vignette contains a series of examples showing how to initialize the safetyGraphics Shiny app in different scenarios. For a general overview of the app see [this vignette](). For more details about adding custom charts, see [this vignette]().
```
Comment (elimillera) [inline vignettes/Cookbook.Rmd:14]: It doesn't look like the 'this vignette' are linking to anything

**Review by elimillera — APPROVED** [review]
Found a couple of typos. Nice work!

File: vignettes/Cookbook.Rmd (hunk 2 above)
Comment (jwildfire) [reply vignettes/Cookbook.Rmd:14]:
```suggestion
This vignette contains a series of examples showing how to initialize the safetyGraphics Shiny app in different scenarios. For a general overview of the app see [this vignette](Intro). For more details about adding custom charts, see [this vignette](chartConfiguration).
```

**Comment by jwildfire** [comment]
Thanks @elimillera. Cleaned up those typos.

**Pull request merged**